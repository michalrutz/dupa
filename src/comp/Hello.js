import React from "react";

class Hello extends React.Component {
    render(){
        return (
            <div>
                <h1>Hello 4!</h1>
                {this.props.children}
            </div>
        )
    }
}
 

export default Hello;
