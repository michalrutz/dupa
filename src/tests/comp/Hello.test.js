import React from "react";
import ReactShallowRendere from "react-test-renderer/shallow";
import Hello from "../../comp/Hello";
import { shallow } from "enzyme";

test("render Hello", ()=> {
    const renderer = new ReactShallowRendere();
    renderer.render(<Hello />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();    
})

test("render Hello", ()=> {
    const wrapper = shallow(<Hello/>);
    expect(wrapper).toMatchSnapshot();
    // expect(wrapper.find("h1").text()).toBe("Hello")
})

