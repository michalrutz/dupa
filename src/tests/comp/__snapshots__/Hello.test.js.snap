// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Hello 1`] = `
<div>
  <h1>
    Hello 4!
  </h1>
</div>
`;

exports[`render Hello 2`] = `
<div>
  <h1>
    Hello 4!
  </h1>
</div>
`;
